spi/src/test/java/org/opensearch/jobscheduler/spi/schedule/CronScheduleTests.java

```diff
@@ -38,16 +38,12 @@ public void setup() {
     }
 
     public void testDifferentClocks() {
-        Clock pdtClock = Clock.system(ZoneId.of("America/Los_Angeles"));
-        Clock utcClock = Clock.system(ZoneId.of("UTC"));
         CronSchedule pdtClockCronSchedule = new CronSchedule("* * * * *", ZoneId.systemDefault());
-        pdtClockCronSchedule.setClock(pdtClock);
         CronSchedule utcClockCronSchedule = new CronSchedule("* * * * *", ZoneId.systemDefault());
-        utcClockCronSchedule.setClock(utcClock);
-        Instant now = Instant.now();
+        Instant instant = Instant.ofEpochSecond(1539715678L);
         assertEquals("Next execution time based on different clock should be same.",
-            pdtClockCronSchedule.getNextExecutionTime(now),
-            utcClockCronSchedule.getNextExecutionTime(now));
+            pdtClockCronSchedule.getNextExecutionTime(instant.atZone(ZoneId.of("America/Los_Angeles")).toInstant()),
+            utcClockCronSchedule.getNextExecutionTime(instant.atZone(ZoneId.of("UTC")).toInstant()));
```

Comment:
I prefer the old test. This seems like it only tests if `instant.atZone(ZoneId.of("America/Los_Angeles")).toInstant()` is equal to `instant.atZone(ZoneId.of("UTC")).toInstant())` which is not what we are trying to test. This modification makes both `pdtClockCronSchedule` and `utcClockCronSchedule` as they are using the same timezone.

Reply:
Good point. Refactored the test.